src/IssueViz.Security/Taint/TaintResources.resx

```diff
@@ -137,6 +137,6 @@
   </data>
   <data name="Synchronizer_UnsupportedSQVersion" xml:space="preserve">
     <value>[Taint] Displaying taint vulnerabilities in the IDE requires SonarQube v8.6 or later, or SonarCloud. Connected SonarQube version: v{0}
-    Visit https://github.com/SonarSource/sonarlint-visualstudio/wiki to find out more about this and other SonarLint features.</value>
+    Visit https://docs.sonarsource.com/sonarlint/visual-studio/using-sonarlint/taint-vulnerabilities/ to find out more about this and other SonarLint features.</value>
```

Comment:
I got a bit confused here as in the text it says we need SonarQube v8.6 or later however in the documentation here -> https://docs.sonarsource.com/sonarlint/visual-studio/using-sonarlint/taint-vulnerabilities/ it says v8.9 or higher. Is that a mistake on our side or?

Reply:
Hhmm. It looks from the code that we support the API call from [v8.6](https://github.com/SonarSource/sonarlint-visualstudio/blob/753f8b7275d3f9d94355603ee373b79e37d40e48/src/SonarQube.Client/Api/DefaultConfiguration.cs#L56) onwards.
I had a quick look in the code but couldn't find any checks against 8.9, so it looks like it should be 8.6.

Reply:
For future reference, we have [a version support policy](https://sonar-documentations-preview.netlify.app/sonarlint/visual-studio/team-features/connected-mode/#sonarlint-sonarqube-version-support-policy) which states we support v8.9+ until sometime around Dec 2023, then only back to the latest LTS.

Reply:
FYI SLVS currently works with older SQ releases. There is a ticket to remove support for older versions: #3665.